sdks/java/io/elasticsearch/pom.xml

```diff
@@ -86,18 +87,19 @@
       <version>4.5.2</version>
     </dependency>
 
-    <dependency>
-      <groupId>joda-time</groupId>
-      <artifactId>joda-time</artifactId>
-    </dependency>
-
     <!-- compile dependencies -->
     <dependency>
       <groupId>com.google.auto.value</groupId>
       <artifactId>auto-value</artifactId>
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>joda-time</groupId>
+      <artifactId>joda-time</artifactId>
+      <scope>provided</scope>
```

Comment:
Why provided? I do not believe we necessarily require runners to provide the `joda-time` jar.